mautic/mautic
# Show overflow in the App Wrapper when a Chosen dropdown is shown

| Q  | A
| --- | ---
| Bug fix? | UI Fix
| New feature? | No
| Related user documentation PR URL | None
| Related developer documentation PR URL | None
| Issues addressed (#s or URLs) | #1706
| BC breaks? | No
| Deprecations? | No

#### Description:
Since the App Wrapper element has its overflow hidden, it prevented the user from seeing all the options of a *Chosen* dropdown when it was at the bottom of the screen (see associated issue).

The Chosen + Bootstrap + `overflow: hidden;` combo seems to be a bit of a controversial issue so this PR attempts to fix it in the simplest, cleanest possible way, even though it *is* kind of a hack.

I wasn't sure **where** to actually write this fix, so I'm hoping I can get some feedback on where I should put my code.

#### Steps to test this PR:
1. Create a new contact
2. At the bottom of the page select an *Owner* (This will be even more apparent if there are many users)
3. Check that the overflow temporarily becomes visible until the dropdown is closed

#### Steps to reproduce the bug:
1. Create a new contact
2. At the bottom of the page select an *Owner* (This will be even more apparent if there are many users)
3. See the cut off UI
